### DISCUSSION TITLE
关于设置初始窗口大小的问题

### DISCUSSION THREAD

**Discussion by Landius** [hooke007/mpv_PlayKit#228] [Q&A 问答]
屏幕分辨率是 3840x2160 的，在 mpv.conf 里设置了 `geometry=50%x50%`。
启动 mpv 后，窗口却不是 1920x1080，而是 1920x1050 的大小。

测了一下，1050 刚好是 2160（屏幕高度） 减去 60（任务栏高度） 的 50%。
难道 `geometry` 计算高度的时候不把任务栏算进去的吗

ps: 用的是 shichiro 0.35 版本 https://sourceforge.net/projects/mpv-player-windows/files/release/mpv-0.35.0-x86_64.7z/download

**Comment by hooke007** [comment]
<s>试试无边框模式。</s>无效
另外geometry的表现受桌面环境影响，百分比在不同平台的表现也不一致，建议直接用具体数值

**Reply by Landius** [reply]
现在就是无边框模式的 `border=no`。
按理说计算高度应该按屏幕高度来算的吧😕

**Reply by hooke007** [reply]
那你只能去mpv的issue tracker提问了（
从注释来看是桌面环境干预的结果
https://github.com/mpv-player/mpv/blob/c2dec023b79ebb54160b5a7a1c5f2b609627f167/video/out/win_state.c#L76-L79
